crypto/src/mls/client/key_package.rs

```diff
@@ -161,10 +162,15 @@ impl Client {
         backend: &MlsCryptoProvider,
         ciphersuite: MlsCiphersuite,
         credential_type: MlsCredentialType,
-    ) -> CryptoResult<usize> {
-        let kps: Vec<MlsKeyPackage> = backend.key_store().find_all(EntityFindParams::default()).await?;
+    ) -> Result<usize> {
+        let kps: Vec<MlsKeyPackage> = backend
+            .key_store()
+            .find_all(EntityFindParams::default())
+            .await
+            .map_err(Error::keystore("finding all key stores"))?;
```

Comment:
Not sure if I get what you mean, but shouldn't this be "finding all keypackages"?

Reply:
Yes, agree. Updating.